amida-tech/orange-api
# quick mood severity field add

adds a field to journal entries to represent the severity of a mood (perhaps a better word than severity should be used on the front end now that I think about it - "intensity" maybe?)

To test:
---
http://amida-tech.github.io/orange-api/#journal-entries
* using Postman, create a journal entry with a moodSeverity
* GET the journal entry
* ensure the returned entry contains the moodSeverity
---
* use orange-ignite branch `feature/quick-moods` to create and view moods
